valkey.conf

```diff
@@ -413,6 +413,14 @@ proc-title-template "{title} {listen-addr} {server-mode}"
 # is derived from the environment variables.
 locale-collate ""
 
+# Valkey is largely compatible with Redis OSS, apart from a few cases where
+# Valkey identifies itself itself as "Valkey" rather than "Redis". Extended
+# Redis OSS compatibility mode makes Valkey pretend to be Redis. Enable this
+# only if you have problems with tools or clients. This is a temporary
+# configuration added in Valkey 8.0 and is scheduled for removal in Valkey 9.0.
```

Comment:
maybe we should also mention the 10.0 thing?:
```
        # This config is added in Valkey 8.0, shall be deprecated and have no
        # effect in 9.x and be deleted in 10.0.
```

Reply:
Good point. "Deprecated" means that it still works, so I think I want to avoid this word.

```suggestion
# configuration added in Valkey 8.0 and is scheduled to have no effect in
# Valkey 9.0 and be completely removed in Valkey 10.0.
```